### PR TITLE
main,bind,_examples: Use absolute package imports and make configurable

### PR DESCRIPTION
**Pull request by justinfx**
This CL fixes an issue with the way python import statements are generated using a relative current directory approach that is no longer compatible with python3. The default behavior has been updated to default to a dot-relative absolute package import for "go" and compiled extension, and a configuration option has been exposed from the CLI flags down to the bind package.
A refactor of configuration options was added to avoid further expanding the number of arguments being passed through the system.

Includes #244 enum import refactor

Fixes go-python/gopy#239

### CODE REVIEW

**Comment by justinfx** [comment]
Rebased to master to fix merge conflicts

**Comment by justinfx** [comment]
This MR is a bit dirty since the fixes from #241 were merge squashed to master. Will fixup now...

File: bind/bind.go
Comment (sbinet) [inline bind/bind.go:19]:
```suggestion
	// name of output package (otherwise name of first package is used)
```

File: bind/bind.go
Comment (sbinet) [inline bind/bind.go:26]:
```suggestion
	VM string
```
(to follow Go's take on acronyms like `id/ID` vs `Id`...)

**Review by sbinet — APPROVED** [review]
LGTM.

thanks.

(I'll wait for #244 to be merged before merging that one as well.)

File: bind/bind.go
Comment (justinfx) [reply bind/bind.go]: Done: 93b1b94

**Comment by sbinet** [comment]
could you rebase off the latest master?
I've migrated to GHA (and dropped Travis-CI), this should speed up the CI turn over time by a lot.

**Comment by justinfx** [comment]
I've rebased but it looks like the windows tests are still appveyor? Is moving those to GHA still wip?

**Comment by sbinet** [comment]
I've indeed just migrated the Linux tests to GHA (as they were the ones in need of some oomph)

**Comment by rcoreilly** [comment]
probably not intentional -- can you submit a PR to fix?

**Pull request merged**
